src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -1039,11 +1112,11 @@ def test_fw_mixed_operator_precedence(fort_writer):
     # Generate Fortran from the PSyIR schedule
     result = fort_writer(schedule)
     expected = (
-        "  a=-a * (-b + c)\n"
-        "  a=-a * (-b + c)\n"
-        "  a=-a + (-b + -c)\n"
-        "  e=.NOT.f .OR. .NOT.g\n"
-        "  a=LOG(b * c)\n")
+        "  a = -a * (-b + c)\n"
+        "  a = -a * (-b + c)\n"
+        "  a = -a + (-b + -c)\n"
+        "  e = .NOT.f .OR. .NOT.g\n"
+        "  a = LOG(b * c)\n")
```

Comment:
Please could you add a compilation check to this test.

Reply:
Done